spinal_levels/README.md

```diff
@@ -0,0 +1,78 @@
+### PAM50 Levels
+
+These level files are a slightly modified copy of the [level files](https://github.com/PhillipsLab/pam50/tree/main/Spinal%20Cord%20Levels%20NIfTI) produced by the [Phillips Lab](https://github.com/PhillipsLab): 
```

Comment:
## Are these the correct level files to use?

> > > Is that normal that in the new version, values seem almost binary, ie: the same value is spread out along the S-I axis, without "smoothness" to it. Eg: zoomed version of C2_new (same value across ~20 voxels). Maybe we should inquired the Phillips group about that?
> >
> >  I'll also check the https://github.com/PhillipsLab/pam50/tree/main/DREZ%20NIfTI to see if the Spinal Cord levels was teh right folder to take here.
>
> I think we should include the Phillips group right away in the conversation to avoid any confusion. I'll take care of that.

Reply:
The response from Aaron Phillips was as follows:

> Q: are these files the latest version and the ones we should for the PAM50?
> A: Yes

So, I'm marking this as resolved.